frontend/src/core/interceptors/error.interceptor.ts

```diff
@@ -30,7 +30,7 @@ export class ErrorInterceptor implements HttpInterceptor {
         },
 
         (err: HttpErrorResponse) => {
-          this.notificationService.error(err.error);
+          this.notificationService.error(err.status + ': ' + err.statusText);
```

Comment:
We need to use error handler and not interceptor, but fine for now

Reply:
Will decide later how to deal with errors in next week